tests/unit/sftp_test.py

```diff
@@ -2,6 +2,8 @@
 
 """Tests suite for sftp."""
 
+import random
+import string
 import uuid
```

Comment:
With https://github.com/ansible/pylibssh/pull/638/files#r1927502223, this import is no longer needed:
```suggestion
```

Reply:
I kept the uuid in the `other_payload()` as I would need the payload for existing files.